srcpkgs/gns3-gui/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'gns3-gui'
 pkgname=gns3-gui
-version=2.2.27
+version=2.2.28
 revision=1
 build_style=python3-module
 # Skip tests/test_link.py::test_create_link test that wasn't updated with the code https://github.com/GNS3/gns3-gui/commit/545a9f53
```

Comment:
This should be fixed now

Reply:
I enabled the test and it failed again.  I don't see any changes to gns3-gui related to those lines in the commit I highlighted.  I should raise an issue upstream.

Reply:
https://github.com/GNS3/gns3-gui/pull/3273 and it's already merged! I should have made that PR a while ago.

I'm debating whether to replicate the change in a patch or continue to skip the test until 2.2.29.

Reply:
Thanks, that's entirely up to you, both solutions sound great.

Reply:
I'll leave it the way it is.  I'll remove the `make_check_args` once the fix is in a released version.